cdap-validation/bin/input.py

```diff
@@ -0,0 +1,66 @@
+
+import getopt
+import sys
+import test_helpers as helpers
+
+##### INPUT FUNCTIONS #####
+
+def process_input(argv):
+## Calls the functions below
+# take input
+#    print blah
+    input_vars = process_params(argv)
+    return input_vars
+
+def process_params(params):
+    input_vars = {}
+    input_vars['verbose'] = 0
+    try:
+        opts, args = getopt.getopt(params, 'hc:m:u:U:vd', ['help', 'cluster=', 'module=', 'user=', 'uri=', 'verbose'])
```

Comment:
would use optparse instead. Better for parsing arguments and generating a usage string